GOVERNANCE.md

```diff
@@ -69,6 +99,49 @@ The Technical Committee meets regularly and the meetings are intended to be open
 Quorum for Technical Committee meetings requires at least a majority of all voting members of the Technical Committee to be present.
 The Technical Committee  may continue to meet if quorum is not met, but will be prevented from making any decisions at the meeting.
 
+### Technical Committee Elections and Voting
+
+Representatives to Eiffel Community Technical Committee are elected by their peers in the Eiffel Community on the basis of merit.
+
+#### Election Rules
+
+Anyone who is a maintainer of one or more projects hosted by Eiffel Community is eligible to run for a seat in Eiffel Technical Committee or nominate another maintainer as a candidate.
+
+Anyone who is a contributor to one or more projects hosted by Eiffel Community is eligible to vote on Technical Committee Elections.
+
+Community elections are overseen by [Election Officers](#Election-Officers).
+
+#### Election Process
+
+The election process consists of 3 phases
+
+* **Preparation Phase**:
+  * Election Officers determine the list of community members who are eligible to **run** in community elections announce this on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+  * Election Officers determine the list community members who are eligible to **vote** in community elections and announce this on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+* **Nomination Phase**:
+  * Once the electorate and candidates are determined and notified, Election Officers start nomination period for Technical Committee elections by sending an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) Nomination period is limited to **one week**.
+  * After the commencement of the nomination phase, eligible candidates who wish to run for a seat in Technical Committee announce their nomination by sending an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community). (self-nomination) If community members want to nominate someone else to Technical Committee, they can do so by sending nomination mail to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) but this must then be accepted by the nominated community member by responding to their nomination mail in order for their candidacy to be valid. Only maintainers can self-nominate or nominate other maintainers.
```

Comment:
I'm a but unclear on what is meant by the "Only maintainers can..." sentence. We previously stated that one has to be a maintainer to be eligible, so... is this redundant? Or is it saying something else that I'm missing? If it's redundant it should be removed, otherwise it needs to be clarified.

Reply:
I agree.

So, the previous sentence states "only maintainers can run in election". This sentence states that as well as who can nominate other maintainers.

Would it be better to replace it with "Only maintainers can nominate other maintainers."?

Reply:
I would suggest simple saying "Only maintainers may nominate candidates."